contracts/src/proxy/IProxy.cairo

```diff
@@ -0,0 +1,32 @@
+# SPDX-License-Identifier: MIT
+# OpenZeppelin Contracts for Cairo v0.3.1 (upgrades/presets/Proxy.cairo)
+
+%lang starknet
```

Comment:
Can we put this within an OZ folder using protostar and git submodule to manage?